renderer.js

```diff
@@ -1147,6 +1157,35 @@ class SDFGRenderer {
         box_select_btn.title = 'Enter box select mode';
         this.toolbar.appendChild(box_select_btn);
 
+        // Enable the static memory overlay
+        let memory_volume_overlay_btn = document.createElement('button');
```

Comment:
I disagree it should be another button. It's getting too populated.

Instead, perhaps have an Overlays sub-menu in the menu

Reply:
Good idea, I'll create an overlays sub-menu and move the option in there.